src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -1633,8 +1628,8 @@ public void installTinyTeX()
       };
 
       dependencyManager_.withTinyTeX(
-            "Installing tinytex",
-            "Installing TinyTeX",
+            constants_.installTinyTeX(),
```

Comment:
There are two different strings here in the original (different capitalization). In the interest of minimizing visible changes in English, please match this with two separate strings.